src/main/java/org/mockito/junit/MockitoJUnit.java

```diff
@@ -33,6 +35,19 @@ public static MockitoRule rule() {
         return new JUnitRule(Plugins.getMockitoLogger(), Strictness.WARN);
     }
 
+    /**
+     * Creates a rule instance that initiates &#064;Mocks and is a {@link TestRule}. Use this method
+     * if you need to explicitly need a {@link TestRule}, for example if you need to compose
+     * multiple rules using a {@link org.junit.rules.RuleChain}.
+     * See {@link MockitoRule}.
+     *
+     * @param testInstance The instance to initiate mocks for
+     * @return the rule instance
+     */
+    public static MockitoTestRule rule(Object testInstance) {
```

Comment:
Can we rename to "testRule"? It is a bit not intuitive when overloaded method returns a different type.

Also, the identical test method name may lead to inconsistencies - some tests using ```rule(this)``` and some ```rule()``` with no good reason.

Reply:
Done